packages/app/components/Schedule/SchedulePanel.tsx

```diff
@@ -32,6 +52,29 @@ export default function SchedulePanel({
       title: e.title,
     })) ?? [];
 
+  const renderButton = () => {
```

Comment:
Could we move this whole button to its own component? It adds 3 hooks to this component so it can clutter the rest of the schedulepanel

Reply:
done